starter_templates/redis/rust/spawn_redis_server.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec cargo run \
```

Comment:
Why is there a `\` at the end of this file?

Reply:
Sir actually there was warning showing in my file that end line should be specified but I should removed it as it was only warning from github

Reply:
Hmm, could you show me a screenshot of the warning? I don't recall seeing that before..

Reply:
Sir it is resolved now. It was showing previously but now it is okay. 

![image](https://user-images.githubusercontent.com/20126545/84154620-c4a67300-aa80-11ea-987d-b3548a19d576.png)